Crash report " java.lang.RuntimeException: Unable to destroy activity {com.github.k1rakishou.chan.beta/com.github.adamantcheese.chan.StartActivity}: kotlin.UninitializedPropertyAccessException: lateinit property updateManager has not been initialized"

UUID: a89ae5f8c29fa29111270384c8826016
Build flavor: beta
Version name: v0.1.1
OS info: Android 9, sdk version: 28
Report description: No title
Logs:
```
=== LOGS(2020-09-02 01:39:30 UTC) ===
W/KurobaEx-beta | StartActivity: Workaround for intent mismatch.
D/KurobaEx-beta | Chan: ^^^ App went foreground ^^^
D/KurobaEx-beta | StartActivity: start
D/KurobaEx-beta | ChanThreadLoader: Requested thread /s/, 2849XXX
D/KurobaEx-beta | PageRequestManager: Requesting existing board pages for /s/, timeout
D/KurobaEx-beta | PageRequestManager: Got pages for 2ch.hk/s/
D/KurobaEx-beta | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() called, isCalledFromForeground=true, isUpdatingCurrentlyOpenedThread=true
D/KurobaEx-beta | LastPageNotificationsHelper: showOrUpdateNotifications(1)
D/KurobaEx-beta | LastPageNotificationsHelper: ChanSettings.watchLastPageNotify is disabled
D/KurobaEx-beta | FetchThreadBookmarkInfoUseCase: FetchThreadBookmarkInfoUseCase.execute(1)
E/KurobaEx-beta | APP: Unhandled exception
E/KurobaEx-beta | APP: java.lang.RuntimeException: Unable to destroy activity {com.github.k1rakishou.chan.beta/com.github.adamantcheese.chan.StartActivity}: kotlin.UninitializedPropertyAccessException: lateinit property updateManager has not been initialized
E/KurobaEx-beta | APP: 	at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4489)
E/KurobaEx-beta | APP: 	at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:4507)
E/KurobaEx-beta | APP: 	at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:39)
E/KurobaEx-beta | APP: 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
E/KurobaEx-beta | APP: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
E/KurobaEx-beta | APP: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1836)
E/KurobaEx-beta | APP: 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/KurobaEx-beta | APP: 	at android.os.Looper.loop(Looper.java:193)
E/KurobaEx-beta | APP: 	at android.app.ActivityThread.main(ActivityThread.java:6702)
E/KurobaEx-beta | APP: 	at java.lang.reflect.Method.invoke(Native Method)
E/KurobaEx-beta | APP: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
E/KurobaEx-beta | APP: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:911)
E/KurobaEx-beta | APP: Caused by: kotlin.UninitializedPropertyAccessException: lateinit property updateManager has not been initialized
E/KurobaEx-beta | APP: 	at com.github.adamantcheese.chan.StartActivity.onDestroy(StartActivity.kt:149)
E/KurobaEx-beta | APP: 	at android.app.Activity.performDestroy(Activity.java:7395)
E/KurobaEx-beta | APP: 	at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1306)
E/KurobaEx-beta | APP: 	at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4474)
E/KurobaEx-beta | APP: 	... 11 more
E/KurobaEx-beta | UNCAUGHT: Called from unhandled exception handler.
E/KurobaEx-beta | UNCAUGHT: java.lang.RuntimeException: Unable to destroy activity {com.github.k1rakishou.chan.beta/com.github.adamantcheese.chan.StartActivity}: kotlin.UninitializedPropertyAccessException: lateinit property updateManager has not been initialized
E/KurobaEx-beta | UNCAUGHT: 	at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4489)
E/KurobaEx-beta | UNCAUGHT: 	at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:4507)
E/KurobaEx-beta | UNCAUGHT: 	at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:39)
E/KurobaEx-beta | UNCAUGHT: 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
E/KurobaEx-beta | UNCAUGHT: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
E/KurobaEx-beta | UNCAUGHT: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1836)
E/KurobaEx-beta | UNCAUGHT: 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/KurobaEx-beta | UNCAUGHT: 	at android.os.Looper.loop(Looper.java:193)
E/KurobaEx-beta | UNCAUGHT: 	at android.app.ActivityThread.main(ActivityThread.java:6702)
E/KurobaEx-beta | UNCAUGHT: 	at java.lang.reflect.Method.invoke(Native Method)
E/KurobaEx-beta | UNCAUGHT: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
E/KurobaEx-beta | UNCAUGHT: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:911)
E/KurobaEx-beta | UNCAUGHT: Caused by: kotlin.UninitializedPropertyAccessException: lateinit property updateManager has not been initialized
E/KurobaEx-beta | UNCAUGHT: 	at com.github.adamantcheese.chan.StartActivity.onDestroy(StartActivity.kt:149)
E/KurobaEx-beta | UNCAUGHT: 	at android.app.Activity.performDestroy(Activity.java:7395)
E/KurobaEx-beta | UNCAUGHT: 	at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1306)
E/KurobaEx-beta | UNCAUGHT: 	at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4474)
E/KurobaEx-beta | UNCAUGHT: 	... 11 more
E/KurobaEx-beta | UNCAUGHT: ------------------------------
E/KurobaEx-beta | UNCAUGHT: END OF CURRENT RUNTIME MESSAGES
E/KurobaEx-beta | UNCAUGHT: ------------------------------
E/KurobaEx-beta | UNCAUGHT: Android API Level: 28
E/KurobaEx-beta | UNCAUGHT: App Version: v0.1.1.772
D/KurobaEx-beta | AndroidUtils: Build Signature: 86242978CF53C34361A8C962D0A57107AEB70E10631AE13EB5B006C0CF673FA9
E/KurobaEx-beta | UNCAUGHT: Development Build: Release


=== SETTINGS ===
Prefetching enabled: false
Hi-res thumbnails enabled: false
Youtube titles and durations parsing enabled: true
Concurrent file loading chunks count: 2
WEBM streaming enabled: true
Saved files base dir info: Java API location: "/storage/emulated/0/KurobaEx-beta/files", SAF location: "", active: Java API
Phone layout mode: AUTO
OkHttp IPv6 support enabled: false
OkHttp HTTP/2 support enabled: true
```

--- a/Kuroba/app/src/main/java/com/github/adamantcheese/chan/StartActivity.kt
+++ b/Kuroba/app/src/main/java/com/github/adamantcheese/chan/StartActivity.kt
@@ -146,10 +146,22 @@ class StartActivity : AppCompatActivity(),
 
     compositeDisposable.clear()
     job.cancel()
-    updateManager.onDestroy()
-    imagePickDelegate.onDestroy()
-    fileChooser.removeCallbacks()
-    globalWindowInsetsManager.requestInsetsApplyFunc = null
+
+    if (::updateManager.isInitialized) {
+      updateManager.onDestroy()
+    }
+
+    if (::imagePickDelegate.isInitialized) {
+      imagePickDelegate.onDestroy()
+    }
+
+    if (::fileChooser.isInitialized) {
+      fileChooser.removeCallbacks()
+    }
+
+    if (::globalWindowInsetsManager.isInitialized) {
+      globalWindowInsetsManager.requestInsetsApplyFunc = null
+    }
 
     while (!stack.isEmpty()) {
       val controller = stack.pop()

